processing/data_collection/gazette/spiders/mg_itauna.py

```diff
@@ -0,0 +1,36 @@
+import datetime as dt
+from dateparser import parse
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgItaunaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3133808"
+    name = "mg_itauna"
+    allowed_domains = ["itauna.mg.gov.br"]
+    start_urls = ["https://www.itauna.mg.gov.br/portal/diario-oficial/"]
+
+    def parse(self, response):
+        url = self.start_urls[0] + "{i}/0/0/0/0/"
```

Comment:
**suggestion**:
It is much safer to use `response.urljoin` to format new URLs instead of concatenating strings manually.
`url = response.urljoin("{i}/0/0/0/0/")` will produce the same result.

**nitpick**: The use of one character variables names decrease the readability of the code, as we don't know exactly what that variable contains. For example, `i` doesn't provide me context. `url = response.urljoin("{page}/0/0/0/0/")` shows the intent of that variable, so we know that every time we format a string, that value will referencing the page number that we want to retrieve.

**nitpick**: This is much more personal, so fell free to ignore as well (as all nitpicks can be ignored). I would prefer to name this variable as `page_url`, to make it clear that this is the URL of a page.